nvflare/app_common/utils/fl_model_utils.py

```diff
@@ -97,7 +97,11 @@ def from_shareable(shareable: Shareable, fl_ctx: Optional[FLContext] = None) ->
         params = None
         meta = {}
 
-        try:
+        submit_model_name = shareable.get_header(AppConstants.SUBMIT_MODEL_NAME)
```

Comment:
There are 2 places where we set this header:

1. https://github.com/NVIDIA/NVFlare/blob/main/nvflare/app_common/ccwf/cse_client_ctl.py#L220-L222
2. https://github.com/NVIDIA/NVFlare/blob/main/nvflare/app_common/workflows/cross_site_model_eval.py#L198-L199

We can see in these cases, since the task is "submit_model", the input is empty (no model params, no metrics)
It is a request ask to send model back.

submit_model does not equal to cross-site evaluation/validation.
submit_model task only happens in cross-site evaluation/validation now.

I could update the comments to say this is "submit_model" task.